ansible/lab

```diff
@@ -118,7 +119,20 @@ sonic_multi_asic:
   vars:
     hwsku: msft_multi_asic_vs
     iface_speed: 40000
+    num_asics: 6
+    start_topo_service: True  
   hosts:
     vlab-07:
-      ansible_host: 10.250.0.109
-      ansible_hostv6: fec0::ffff:afa:7
+      ansible_host: 10.250.0.110
+      ansible_hostv6: fec0::ffff:afa:a
+
+sonic_multi_asic_2:
+  vars:
+    hwsku: msft_four_asic_vs
+    iface_speed: 40000
+    num_asics: 4
```

Comment:
Please add the frontend asic attribute as well 
`frontend_asics: [2,3]`

Reply:
Updated to [0,1].